stub-lite/build.gradle

```diff
@@ -29,10 +26,12 @@ dependencies {
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:${rootProject.ext.coroutinesVersion}"
 
     // Grpc and Protobuf
-    implementation "io.grpc:grpc-protobuf-lite:${rootProject.ext.grpcVersion}"
-    implementation "io.grpc:grpc-stub:${rootProject.ext.grpcVersion}"
-    implementation ("io.grpc:grpc-kotlin-stub:$version") {
-        exclude group: 'io.grpc', module: 'grpc-protobuf'
+    api "io.grpc:grpc-protobuf-lite:${rootProject.ext.grpcVersion}"
+    api "com.google.guava:guava:29.0-android"
```

Comment:
This feels wrong but if I don't include it, in the `examples` I get an error. To repro from this branch:

Change to the local maven and remove the dep on `com.google.guava:guava:29.0-android`:
```
diff --git a/examples/build.gradle.kts b/examples/build.gradle.kts
index 5d78d5d..e4150aa 100644
--- a/examples/build.gradle.kts
+++ b/examples/build.gradle.kts
@@ -7,7 +7,7 @@ plugins {
 
 // todo: move to subprojects, but how?
 ext["grpcVersion"] = "1.32.1"
-ext["grpcKotlinVersion"] = "0.2.0" // CURRENT_GRPC_KOTLIN_VERSION
+ext["grpcKotlinVersion"] = "0.2.1-SNAPSHOT" // CURRENT_GRPC_KOTLIN_VERSION
 ext["protobufVersion"] = "3.13.0"
 
 allprojects {
diff --git a/examples/stub-lite/build.gradle.kts b/examples/stub-lite/build.gradle.kts
index aacd6ed..3e676cb 100644
--- a/examples/stub-lite/build.gradle.kts
+++ b/examples/stub-lite/build.gradle.kts
@@ -20,13 +20,15 @@ dependencies {
     api("io.grpc:grpc-protobuf-lite:${rootProject.ext["grpcVersion"]}")
 
     // todo: I think we should be able to just include this:
-    // api("io.grpc:grpc-kotlin-stub-lite:${rootProject.ext["grpcKotlinVersion"]}")
+    api("io.grpc:grpc-kotlin-stub-lite:${rootProject.ext["grpcKotlinVersion"]}")
     // but it doesn't pull transitives correctly
 
+    /*
     api("io.grpc:grpc-stub:${rootProject.ext["grpcVersion"]}")
     api("io.grpc:grpc-kotlin-stub:${rootProject.ext["grpcKotlinVersion"]}") {
         exclude("io.grpc", "grpc-protobuf")
     }
+     */
 }
 
 java {
diff --git a/stub-lite/build.gradle b/stub-lite/build.gradle
index 3274de7..79798f2 100644
--- a/stub-lite/build.gradle
+++ b/stub-lite/build.gradle
@@ -27,7 +27,7 @@ dependencies {
 
     // Grpc and Protobuf
     api "io.grpc:grpc-protobuf-lite:${rootProject.ext.grpcVersion}"
-    api "com.google.guava:guava:29.0-android"
+    //api "com.google.guava:guava:29.0-android"
     api "io.grpc:grpc-stub:${rootProject.ext.grpcVersion}"
 
     api(project(":grpc-kotlin-stub")) {
```

Then publish `0.2.1-SNAPSHOT` locally:
```
./gradlew publishToMavenLocal
```

Now try to build the examples:
```
cd examples
./gradlew :native-client:run
```

Error is:
```
/home/jamesward/projects/grpc-kotlin/examples/stub-lite/build/generated/source/proto/main/grpc/io/grpc/examples/routeguide/RouteGuideGrpc.java:433: error: package com.google.common.util.concurrent does not exist
    public com.google.common.util.concurrent.ListenableFuture<io.grpc.examples.routeguide.Feature> getFeature(
```

Reply:
@saturnism Can you help with this?